packages/components/Footer.tsx

```diff
@@ -1,3 +1,96 @@
+        fill="#2F855A"
+      />
+    </svg>
+  );
+};
+
+const ListHeader = ({ children }: { children: ReactNode }) => {
+  return (
+    <Text fontWeight="500" fontSize="lg" mb={2}>
+      {children}
+    </Text>
+  );
+};
+
 export default function Footer() {
-  return <>This is the Footer component</>;
+  return (
+    <Box
+      bg={useColorModeValue('gray.50', 'gray.900')}
+      color={useColorModeValue('gray.700', 'gray.200')}
+      borderTop="2px"
+    >
+      <Container as={Stack} maxW="6xl" py={10}>
+        <SimpleGrid
+          templateColumns={{ sm: '1fr 1fr', md: '2fr 1fr 1fr 1fr 1fr' }}
```

Comment:
muito feio, não não.

usa o repeat do css for favor.

Reply:
Talvez seja bom repensar. Como já disse antes, o flex é melhor pra fazer coisas in-line